pkg/envoy/server.go

```diff
@@ -534,7 +470,7 @@ func UpdateNetworkPolicy(ep NetworkPolicyEndpoint, policy *policy.L4Policy,
 	for _, p := range policies {
 		policyRevision := policy.Revision
 		callback := func() {
-			ep.OnProxyPolicyUpdate(policyRevision)
+			go ep.OnProxyPolicyUpdate(policyRevision)
```

Comment:
Is it worth adding a comment explaining why this starts a goroutine instead of doing it synchronously?